src/wasm-lib/kcl/src/std/extrude.rs

```diff
@@ -181,6 +181,48 @@ pub(crate) async fn do_post_extrude(
         vec![]
     };
 
+    for face_info in face_infos.iter() {
```

Comment:
@jess I wanted to double-check with you about this for two reasons.

1) with this change, it makes these opposite and adjacent edge requests so that we can populate the artifact map. but perhaps we should load this onto the Extrude group as well?

2) leading on from 1) I know that we do some magic with the fillets where we delay them, maybe with this change, knowing all of the edge ids ahead of time means the `getOppositeEdge` etc stdlib functions  can reference edges ids already in the extrude group instead of having to make their own requests and that way it doesn't need to be delayed?

But maybe I have a bad concept of the fillet code.

Reply:
I think jess is busy right now and we need to decide ourselves if this is a good decision. Based on what you've said already it sounds better in terms of separation of concerns and simpler code.

Reply:
Yeah that's fair I might merge this, the question was somewhat future looking anyway because I was proposing a changing how the fillet/getOppositeEdge stdlib functions work.